code/game/objects/structures/mannequin.dm

```diff
@@ -0,0 +1,1386 @@
+	maxHealth = 30
+
+/obj/structure/mannequin/wood/Destroy()
+	getFromPool(/obj/item/stack/sheet/wood, loc, 5)//You get half the materials used to make a block back
+	..()
+
+/obj/structure/mannequin/wood/fat
+	name = "fat human wooden mannequin"
+	icon_state="mannequin_wooden_fat_human"
+	fat = 1
+
+/obj/structure/mannequin/wood/vox
+	name = "vox wooden mannequin"
+	icon_state="mannequin_wooden_vox"
+	species_type = /datum/species/vox
+
+
+
+
+
+/////////////////////////////////////////////////////////BLOCKS//////////////////////////////////////////////////////////
+//Use a chisel on those to sculpt them into mannequins.
+
+/obj/structure/block
+	name = "marble block"
+	desc = "Grab your chisel and get to work!"
+	anchored = 0
+	density = 1
+	icon = 'icons/obj/mannequin.dmi'
+	icon_state = "marble"
+	var/time_to_sculpt = 200
+	var/result = "/obj/structure/mannequin"
+	var/no_drop = 0
+
+/obj/structure/block/attackby(var/obj/item/weapon/W,var/mob/user)
+	if(iswrench(W))
+		return wrenchAnchor(user, 50)
+	else if(istype(W, /obj/item/weapon/chisel))
+
+		var/list/available_sculptures = list(
+			"human",
+			"fat human",
+			"vox",
+			)
+
+		var/chosen_sculpture = input("Choose a sculpture type.", "[name]") as null|anything in available_sculptures
+
+		if(!chosen_sculpture || !Adjacent(user))
+			return
+
+		user.visible_message("[user.name] starts sculpting \the [src] with a passion!","You start sculpting \the [src] with a passion!","You hear a repeated knocking sound.")
+		var/turf/T=get_turf(src)
+
+		if(do_after(user, src, time_to_sculpt))
+			new /obj/effect/decal/cleanable/dirt(T)
+			if(findtext(chosen_sculpture, "vox"))
+				result += "/vox"
+			var/obj/structure/mannequin/M = new result(T)
+			if(findtext(chosen_sculpture, "fat"))
+				M.fat = 1
+			M.name = "[chosen_sculpture] [icon_state] mannequin"
+			M.anchored = anchored
+			M.icon_state = "mannequin_[icon_state]_[replacetext(chosen_sculpture, " ", "_")]"
+			M.add_fingerprint(user)
+			user.visible_message("[user.name] finishes \the [M].","You finish \the [M].")
+			no_drop = 1
+			qdel(src)
+		return 1
+	else
+		..()
+
+/obj/structure/block/wood
+	name = "wooden block"
+	icon_state = "wooden"
+	time_to_sculpt = 100
+	result = "/obj/structure/mannequin/wood"
+
+/obj/structure/block/wood/Destroy()
+	if(!no_drop)
+		getFromPool(/obj/item/stack/sheet/wood, loc, 10)
+	..()
+
+
+
+
+
+
+/////////////////////////////////////////////////////////CYBER MANNEQUIN//////////////////////////////////////////////////////////
+//Mannequin meets Display Case.
+
+/obj/structure/mannequin/cyber
+	name = "human cyber mannequin"
+	desc = "Holy shit."
+	icon = 'icons/obj/mannequin_64x64.dmi'
+	icon_state="mannequin_cyber_human"
+	pixel_x = -16
+	clothing_offset_x = 16
+	clothing_offset_y = 7
+	health = 150
+	maxHealth = 150
+	var/shield = 50
+	var/maxShield = 50
+	var/destroyed = 0
+	var/locked = 0
+
+/obj/structure/mannequin/cyber/New()
+	..()
+	update_icon()
+
+/obj/structure/mannequin/cyber/Destroy()
```

Comment:
Jesus christ this really isn't the point of `Destroy`.

Move all this spawning junk crap to the things that it logically makes sense for, not this.